chapters/hardware-software-interface/x86-architecture-family/drills/tasks/conditional-jumps/README.md

```diff
@@ -0,0 +1,23 @@
+# Conditional jumps
+
+You will solve the exercises starting from the ```hello_world.asm``` file located in the ```drills/tasks/conditional-jumps``` directory.
+
+1. Modify the program so that the message is displayed only if the content of the **eax** register is greater than that of **ebx**. Also, modify the values of the registers to continue displaying the message `"Hello, World!"`.
```

Comment:
```suggestion
1. Modify the program so that the message is displayed only if the content of the `eax` register is greater than that of `ebx`. Also, modify the values of the registers to continue displaying the message `"Hello, World!"`.
```
Use backticks, not bold for any code or paths